WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -588,7 +588,7 @@ def my_competitions
       competition_ids.concat(current_user.person.competitions.pluck(:competitionId))
     end
     competitions = Competition.includes(:delegate_report, :delegates)
-                              .where(id: competition_ids.uniq)
+                              .where(id: competition_ids.uniq).where("cancelled_at is null or end_date >= curdate()")
```

Comment:
why do we need the `end_date >= curdate()` check here?

Reply:
The logic is that an organiser might still have duties to perform for a competition until the date of the competition has passed. For example, mailing all competitors about the cancellation. In general ensuring ease of access until it is certain that they won't need to frequently visit the page anymore.

Reply:
ah ok, can you write a comment about that?

Reply:
Done